src/Containers/LeftSideBar/CollapsibleHeader.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { AngleDown } from '@styled-icons/fa-solid/AngleDown';
+import { StyledIcon } from '@styled-icons/styled-icon';
+import { 
+    Draggable,
+    DraggableProvided,
+    DraggableStateSnapshot
+} from 'react-beautiful-dnd';
+import { ReceiptElements, draggableComponent } from './ReceiptElements';
+import { DraggableElement } from './DraggableElement';
+import { Heading } from '../../Text/Heading';
+import { MainInterface, ResponsiveInterface } from '../../Utils/BaseStyles';
+import { Mixins } from '../../Utils';
+
+export interface CollapsibleHeaderProps extends MainInterface, ResponsiveInterface, React.HTMLAttributes<HTMLDivElement> {
+    padding?: string,
+    headerFlex?: string,
+    icon?: StyledIcon,
+    category: string
+};
+
+interface WrapperProps {
+    padding?: string;
+};
+
+interface RowProps {
+    display?: string;
+};
+
+interface IconProps {
+    isCollapsed?: boolean;
+};
+
+export const CollapsibleHeader: React.FC<CollapsibleHeaderProps> = ({
+    padding = '10px 20px',
+    headerFlex = 'space-between',
+    icon,
+    category,
```

Comment:
Not sure what this means in the context of CollapsibleHeader? Is this headerContents?

Reply:
headerTitle?